### DISCUSSION TITLE
In 1997, David Wolf became the first American to vote from where?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#458] [Galaxy Trivia] [answered]
**Bonus:** Can you name the last person who did their civic duty the way David did?

**Comment by 0x4248** [comment]
David Wolf is an American astronaut who made history in 1997 by becoming the first American to vote in a presidential election while in space.

**Comment by thuongtruong1009** [comment, marked as answer]
In 1997, David Wolf became the first American to vote from space. He was an astronaut on the Russian space station Mir and cast his vote in a local election in Texas using a secure electronic ballot that was sent to him via email. Wolf's vote was transmitted to the county clerk's office in Texas and counted along with other absentee ballots. This was an important milestone in the history of space exploration and demonstrated the feasibility of remote voting for people who are living and working in space for extended periods of time.
